alfasim_sdk/types.py

```diff
@@ -248,3 +246,18 @@ def check(  # pylint: disable=arguments-differ
 @attr.s(kw_only=True)
 class Boolean(BaseField):
     value: bool = attrib(validator=instance_of(bool))
+
+
+@attr.s(kw_only=True)
+class FilePath(BaseField):
+    """
+    The FilePath component provides a platform-native file dialog to the user to be able to select a file.
+    The name of the selected file will be available over the GUI and be enabled to be manually changed.
+
+    If you want to make the file mandatory is recommended to include a status monitor in your plugin
+    to make sure that that a file is selected.
+
+    For more details about status monitor check alfasim_sdk.status.ErrorMessage
+    Properties:
```

Comment:
Hmm this is still here 😕

Reply:
I solved the conversation before the push 😄